binr/r2r/r2r.c

```diff
@@ -730,11 +730,11 @@ static void interact(R2RState *state) {
 		print_result_diff (&state->run_config, result);
 inval:
 		printf ("Wat do?    "
-				"(f)ix \xe2\x9c\x85\xef\xb8\x8f\xef\xb8\x8f\xef\xb8\x8f    "
-				"(i)gnore \xf0\x9f\x99\x88    "
-				"(b)roken \xe2\x98\xa0\xef\xb8\x8f\xef\xb8\x8f\xef\xb8\x8f    "
-				"(c)ommands \xe2\x8c\xa8\xef\xb8\x8f    "
-				"(q)uit \xf0\x9f\x9a\xaa\n");
+				"(f)ix ✅"UTF8_VS16 UTF8_VS16 UTF8_VS16"    "
```

Comment:
what about using a define instead of inlining emojis?

Reply:
ok 604f92cc57d35884dd6119282797896181f5cee3